pkg/logging/encoder.go

```diff
@@ -0,0 +1,73 @@
+package logging
+
+import (
+	"strconv"
+	"time"
+
+	"go.uber.org/zap/zapcore"
+)
+
+var (
+	_ zapcore.ObjectEncoder = (*stringObjectEncoder)(nil)
+)
+
+// stringObjectEncoder encodes zapcore.Fields to a map[string]string for conversion reasons.
+// Currently, following types will not be ignored, since those probably do not make much sense to expose:
+//   - Complex numbers
+//   - Arrays, Objects
+//   - Binary data
+type stringObjectEncoder struct {
+	m map[string]string
+}
+
+func (z *stringObjectEncoder) AddBool(key string, value bool) {
+	z.m[key] = strconv.FormatBool(value)
+}
+func (z *stringObjectEncoder) AddDuration(key string, value time.Duration) {
+	z.m[key] = value.String()
+}
+
+func (z *stringObjectEncoder) AddFloat64(key string, value float64) {
+	z.m[key] = strconv.FormatFloat(value, 'f', -1, 64)
+}
+
+func (z *stringObjectEncoder) AddInt64(key string, value int64) {
+	z.m[key] = strconv.FormatInt(value, 10)
+}
+
+func (z *stringObjectEncoder) AddString(key, value string) {
+	z.m[key] = value
+}
+
+func (z *stringObjectEncoder) AddTime(key string, value time.Time) {
+	z.m[key] = value.String()
```

Comment:
```suggestion
	z.m[key] = value.Format(time.RFC3339Nano)
```
Example:
```
.String() = 2000-02-01 12:13:14.000000015 +0000 UTC
.Format(...) = 2000-02-01T12:13:14.000000015Z
```

Although maybe personal taste I guess.

Reply:
Just had a similar comment in another PR, seems like we settle on `Format(time.RFC3339)`.